### ISSUE TITLE
[Rumble] Added missing RumbleEmbed extractor test keys.

### ISSUE THREAD

**Issue by kwconder** [yt-dlp/yt-dlp#3977]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.05.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Description

The default test for the RumbleEmbed extractor fails.
>python test/test_download.py TestDownload.test_RumbleEmbed

<pre>
AssertionError: {'channel_url', 'thumbnail', 'duration', 'channel'} is not false : Missing keys in test definition: channel, channel_url, duration, thumbnail
</pre>

I created a pull request that adds the missing keys: https://github.com/yt-dlp/yt-dlp/pull/3976

### Verbose log

```shell
[debug] [RumbleEmbed] Extracting URL: https://rumble.com/embed/v5pv5f
[RumbleEmbed] v5pv5f: Downloading JSON metadata
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] v5pv5f: Downloading 1 format(s): mp4-720p
[info] Writing video metadata as JSON to: test_RumbleEmbed_v5pv5f.info.json
[debug] Invoking http downloader on "https://sp.rmbl.ws/s8/2/5/M/z/1/5Mz1a.gaa.1.mp4"
[download] Destination: test_RumbleEmbed_v5pv5f.mp4
[download] 100% of 10.00KiB in 00:00

'info_dict': {
    'id': 'v5pv5f',
    'title': 'WMAR 2 News Latest Headlines | October 20, 6pm',
    'timestamp': 1571611968,
    'upload_date': '20191020',

    'channel': 'WMAR',
    'channel_url': 'https://rumble.com/c/WMAR',
    'thumbnail': 'https://sp.rmbl.ws/s8/1/5/M/z/1/5Mz1a.OvCc-small-WMAR-2-News-Latest-Headline.jpg',
    'duration': 234,
},
F
======================================================================
FAIL: test_RumbleEmbed (__main__.TestDownload):
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_download.py", line 218, in test_template
    expect_info_dict(self, tc_res_dict, tc.get('info_dict', {}))
  File "/home/debug/clone/yt-dlp/test/helper.py", line 258, in expect_info_dict
    self.assertFalse(
AssertionError: {'channel', 'channel_url', 'thumbnail', 'duration'} is not false : Missing keys in test definition: channel, channel_url, duration, thumbnail

----------------------------------------------------------------------
Ran 1 test in 0.794s

FAILED (failures=1)
```

**Comment by gamer191** [comment]
While the PR will likely be merged (I'm not a maintainer), please understand that failing tests aren't really bugs. Once upon a time they probably would have been, but now there are so many failing tests, that the download tests were [disabled last year](https://github.com/yt-dlp/yt-dlp/commit/3638226215215383c7a83517479c5df18ab20ac5) (previously they ran on every commit, now people who submit PRs are expected to manually run only the applicable extractor's tests). In fact, I ran the download tests a few days ago, and you can see the results [here](https://github.com/gamer191/yt-dlp/runs/6687564654?check_suite_focus=true) (click "run tests" to expand it, and then you can see a full list of all the tests and whether they succeeded or failed). I am planning on creating a PR to fix (some of) the generic extractor's failing tests, but that probably won't be ready for ages, and there will still be hundreds of failing site extractor tests (which are also somewhere on my to do list)

**Comment by gamer191** [comment]
That said, thanks for the PR, it is useful (my point was just that it isn't a bug that needs to be reported)

**Comment by pukkandan** [comment]
The PR is fine, but this issue isn't needed

**Comment by gamer191** [comment]
@kwconder Out of interest, why were you running the tests? Are you working on a PR (apart from #3976) which modifies the Rumble extractor?

**Comment by kwconder** [comment]
I was interested in fixing Issue [#2846: Extract description](https://github.com/yt-dlp/yt-dlp/issues/2846). I thought I'd make my first pull request simple so I could understand the process. 

I see Pull Request [#3031: Extract description, view count](https://github.com/yt-dlp/yt-dlp/pull/3031) fixed Issue [#2846](https://github.com/yt-dlp/yt-dlp/issues/2846). I'll wait for that one to be merged.

**Comment by pukkandan** [comment]
> I see Pull Request https://github.com/yt-dlp/yt-dlp/pull/3031 fixed Issue https://github.com/yt-dlp/yt-dlp/issues/2846. I'll wait for that one to be merged.

The PR is closed by its author. Idk if they want to re-open it. You are better of creating a new PR
